Formula/o/orbuculum.rb

```diff
@@ -0,0 +1,47 @@
+class Orbuculum < Formula
+  desc "Arm Cortex-M SWO/SWV Demux and Postprocess"
+  homepage "https://github.com/orbcode/orbuculum"
+  url "https://github.com/orbcode/orbuculum/archive/refs/tags/V2.1.0.tar.gz"
+  sha256 "ccdd86130094001a0ab61e5501a6636e12c82b0b44690795a2911c65c5618c46"
+  license "BSD-3-Clause"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "capstone"
+  depends_on "libusb"
+  depends_on "sdl2"
+  depends_on "zeromq"
+  uses_from_macos "ncurses"
+
+  on_macos do
+    depends_on "libelf"
+  end
+  on_linux do
+    depends_on "elfutils"
+  end
+
+  def install
+    system "meson", "setup", "build", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build", "--tags", "devel,runtime"
+  end
+
+  # The tools do not report their version correctly when installed from a
+  # GitHub release rather than from Git directly for versions <= 2.1.0.
+  # This has now been rectified, and the versions can be tested for future
+  # releases.
+  test do
+    shell_output("#{bin}/orbuculum --version", 255)
+    shell_output("#{bin}/orbcat --version", 255)
+    shell_output("#{bin}/orbdump --version", 255)
+    shell_output("#{bin}/orbfifo --version", 255)
+    shell_output("#{bin}/orblcd --version", 255)
+    shell_output("#{bin}/orbmortem")
+    shell_output("#{bin}/orbprofile", 254)
+    shell_output("#{bin}/orbstat", 254)
+    shell_output("#{bin}/orbtop", 247)
+    shell_output("#{bin}/orbtrace")
+    shell_output("#{bin}/orbzmq --version", 255)
```

Comment:
Please also check for expected output here, using `assert_match "a", shell_output("b")`

Reply:
Done in 38289386fe74e0f367c55d9ce8ce1ee6a0073f65